content/v2.0/router/advanced-usage.md

```diff
@@ -2,3 +2,52 @@
 title: "Router: Advanced Usage"
 order: 30
 ---
+
+
+[Refer](https://github.com/hanami/router/blob/unstable/spec/integration/hanami/router/mount_spec.rb)
+
+## Mounting an application
+
+In Hanami, you can mount any rack application in desired route. If we want to do it, you should use the `mount` keyword.
```

Comment:
This is in the rack configuration, right? `config.ru` rather than a route files?

Reply:
oh, is it? I will check, thanks! I know it was in the config in 1.3, but I think I saw it changed in new tests.

Will get back to it! cc: @jodosha

Reply:
Nope, it's the router.

@swilgosz I strongly advice to wrap those lines into the following block:

```ruby
# config/routes.rb
module MyApp
  class ApplicationRoutes < Hanami::Application::Routes
    mount # ...
  end
end
```

Please be aware that this syntax may change, but at least it's crystal clear of what we're talking about: application routes, and not `config.ru`.

---

I would not use "keyword" but "method" instead. Usually keywords are langs constructs.

Reply:
@jodosha ATM Hanami2APplication template does not inherit from Hanami::Application::Routes, but rather passes a block to the main class

https://github.com/hanami/hanami-2-application-template/blob/main/config/routes.rb